app/styles/partials/popups.scss

```diff
@@ -0,0 +1,28 @@
+.ui.notification.popup {
+  padding: 2px;
+  .list {
+    width: auto;
+    max-height: 50vh;
+    overflow: hidden;
+    overflow-y: scroll;
+    padding: 0;
+    margin: 0;
+    .item .content {
+      .header {
+        color: rgba(0, 0, 0, 0.87);;
```

Comment:
Why the custom color and weight? The moment we start using custom colors like this for event basic elements we loose global theming capability.